code/__defines/misc.dm

```diff
@@ -191,7 +191,6 @@
 #define CELLS 8								//Amount of cells per row/column in grid
 #define CELLSIZE (world.icon_size/CELLS)	//Size of a cell in pixels
 
-#define WORLD_ICON_SIZE 32
```

Comment:
this is very necessary, is it duplicated elsewhere?

Reply:
![image](https://user-images.githubusercontent.com/49106866/120087672-f1033700-c0b7-11eb-8822-1a46b6b74da7.png)